### PR TITLE
Guard against launching multiple Chrome processes

### PR DESCRIPTION
**Pull request by Janpot**

### CODE REVIEW

**Comment by wardpeet** [comment]
LGTM 😄 just a question when did you have this issue? ^^

**Comment by Janpot** [comment]
@wardpeet Not in lighthouse. Subclassing this ChromeLauncher in my own code. Got this issue because I assumed `run` would guard against this behaviour. I was using `run` as a substitute for (`run` if not yet running + `waitUntilReady`) in multiple places.

File: lighthouse-cli/chrome-launcher.ts [hunk 1]
```diff
@@ -136,6 +136,11 @@ class ChromeLauncher {
 
   spawn(execPath: string): Promise<any[]> {
     return new Promise((resolve, reject) => {
+      if (this.chrome) {
+        log.verbose('ChromeLauncher', `Chrome already running with pid ${this.chrome.pid}.`);
```
Comment (brendankenny) [inline lighthouse-cli/chrome-launcher.ts]: maybe use `log.warn`? Seems like an unusual situation that in most cases you'd want to know your use of launcher is trying to launch chrome twice

Comment (brendankenny) [inline lighthouse-cli/chrome-launcher.ts]: (the tradeoff being that it adds warning noise to a use case like your own. A compromise might be just `log.log`)

Comment (Janpot) [inline lighthouse-cli/chrome-launcher.ts]: It's all the same for me, I just took whatever was underneath. Just let me know what you prefer

**Comment by Janpot** [comment]
@brendankenny Changed to `log.log`

**Pull request merged**